### ISSUE TITLE
Verifier: allow relative/absolute line references in anchor specs. [phab:D478]

### ISSUE THREAD

**Issue by kythe** [google/kythe#712]
_Created by **zarko** at 2015-08-29 00:47:39_: 

This change makes it possible to handle code where it is syntactically
impossible to place a goal comment directly before some text containing
interesting substrings. This often comes up when testing indexers
that look at documentation strings, where interposing a goal comment
would break the association between documentation and the thing
being documented:

```
  /// This class does X
  //- X defines XClass
  class X { ... };
```

Now, you can do this:

```
  //- +2"X" defines XClass
  /// This class does X
  class X { ... };
```

Previously you had to resort either to changing the goal comment
indicator or to embedding offsets in tests.

Thanks to schroederc for the suggestion.

**Comment by kythe** [comment]
_**schroederc** accepted this revision at 2015-08-31 15:53:10_

**Comment by kythe** [comment]
_**zarko** closed via commit e2d252c19d23155024300a3441ee6da869121fce at 2015-08-31 16:00:52_
